packages/react-components/src/components/VideoGallery/useVideoTileContextualMenuProps.ts

```diff
@@ -28,6 +28,22 @@ export const useVideoTileContextualMenuProps = (props: {
     stopSpotlightOnSelfVideoTileMenuLabel?: string;
     spotlightLimitReachedMenuTitle?: string;
     muteParticipantMenuItemLabel?: string;
+    /* @conditional-compile-remove(media-access) */
+    forbidAudio?: string;
```

Comment:
These strings should have a suffix to reflect either its position or its action it will take. so if these are to take an action on the participants actions they would be something like `forbidAudioForParticipant` or if a label on a button should be `forbidAudioButtonLabel`

Reply:
Fixed